eng/Versions.props

```diff
@@ -7,4 +7,9 @@
     <VersionPrefix>7.0.0</VersionPrefix>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
   </PropertyGroup>
+  <!--Package versions-->
+  <PropertyGroup>
+    <MicrosoftSourceBuildIntermediatesourcebuildreferencepackagesPackageVersion>8.0.0-alpha.1.22517.1</MicrosoftSourceBuildIntermediatesourcebuildreferencepackagesPackageVersion>
+    <MicrosoftSourceLinkGitHubPackageVersion>1.2.0-beta-22503-03</MicrosoftSourceLinkGitHubPackageVersion>
```

Comment:
@MichaelSimons are these versions actually required? They aren't used anywhere in the repo build.

Reply:
No they aren't.  I don't think they should have been added.  I suspect darc was used to add/update the dependency and auto added these.

Reply:
Cool. Fixed in https://github.com/dotnet/xdt/pull/509